### PR TITLE
Corrected script for some item #15

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

**Comment by Atemo** [comment]
Which server do these items come from? (so that I don't have to search through all the servers)

**Comment by idk-whoami** [comment]
overall from jRO  but not for sale from thRO

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -102037,6 +102227,47 @@ Body:
...
+      .@r= getrefine();
+      bonus bMdef,10;
+      bonus bFlee,10;
+      if (getskilllv("KO_KAIHOU") >= 1)
+         bonus2 bSkillAtk,"KO_KAIHOU",40;
+      if (getskilllv("NJ_HUUMA") >= 5)
+         bonus2 bSkillAtk,"NJ_HUUMA",50;
+      if (getskilllv("KO_HUUMARANKA") >= 5)
+         bonus2 bSkillAtk,"KO_HUUMARANKA",30;
+      if (eaclass()&EAJL_THIRD && BaseJob == Job_KagerouOboro) {
```
Comment (Atemo) [inline db/re/item_db_equip.yml:102253]:
```suggestion
      if (eaclass()&EAJL_THIRD && (BaseJob == Job_Kagerou || BaseJob == Job_Oboro)) {
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -102037,6 +102227,47 @@ Body:
...
+         bonus bMaxHPrate,15;
+         bonus bLongAtkDef,20;
+      }   
+      if (.@r>=7) {
+         bonus bFlee,20;
+         bonus2 bSkillAtk,"NJ_HUUMA",BaseLevel;
+         bonus2 bSkillAtk,"KO_KAIHOU",3*(BaseLevel/2);
+         bonus2 bSkillAtk,"KO_HUUMARANKA",BaseLevel/3;
+         if (eaclass()&EAJL_THIRD && BaseJob == Job_KagerouOboro) {
```
Comment (Atemo) [inline db/re/item_db_equip.yml:102262]:
```suggestion
         if (eaclass()&EAJL_THIRD && (BaseJob == Job_Kagerou || BaseJob == Job_Oboro)) {
```

File: db/re/item_combos.yml [hunk 3]
```diff
@@ -32236,3 +32268,823 @@ Body:
...
+          - Improve_Orb_T_WIND    # 310273
+    Script: |
+      .@r = getequiprefinerycnt(EQI_HEAD_TOP);
+      bonus2 bMagicAtkEle,Ele_Wind,3*.@r;      
+  - Combos:
+      - Combo:
+          - Golden_Scarf_J    # 20900
+          - Imuke_Upper_Shield    # 28910
+    Script: |
+      if (eaclass()&EAJL_THIRD && BaseJob == Job_KagerouOboro)
```
Comment (Atemo) [inline db/re/item_combos.yml:33031]:
```suggestion
      if (eaclass()&EAJL_THIRD && (BaseJob == Job_Kagerou || BaseJob == Job_Oboro))
```

File: db/re/item_combos.yml [hunk 4]
```diff
@@ -32236,3 +32268,823 @@ Body:
...
+         bonus bLongAtkDef,-35;
+  - Combos:
+      - Combo:
+          - Golden_Scarf_J    # 20900
+          - Horn_Card    # 4045
+    Script: |
+      if (eaclass()&EAJL_THIRD && BaseJob == Job_KagerouOboro)
```
Comment (Atemo) [inline db/re/item_combos.yml:33038]:
```suggestion
      if (eaclass()&EAJL_THIRD && (BaseJob == Job_Kagerou || BaseJob == Job_Oboro))
```

File: db/re/item_combos.yml [hunk 5]
```diff
@@ -32236,3 +32268,823 @@ Body:
...
+         bonus bLongAtkDef,-35;
+  - Combos:
+      - Combo:
+          - Golden_Scarf_J    # 20900
+          - Automatic_Orb_J01    # 310653
+    Script: |
+      if (eaclass()&EAJL_THIRD && BaseJob == Job_KagerouOboro)
```
Comment (Atemo) [inline db/re/item_combos.yml:33045]:
```suggestion
      if (eaclass()&EAJL_THIRD && (BaseJob == Job_Kagerou || BaseJob == Job_Oboro))
```

**Review by Atemo — COMMENTED** [review]
Good work!

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:102253]: Thanks!

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:102262]: Thanks!

File: db/re/item_combos.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**